portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -278,6 +283,11 @@
 	<td>//div[contains(@class,'form-container')]//div[contains(@data-field-name,'${key_fieldName}')][${key_repeatableFieldNumber}]//label[contains(.,'${key_fieldLabel}')] | //div[contains(@class,'form-container')]//div[contains(@data-field-name,'${key_fieldName}')][${key_repeatableFieldNumber}]//p[contains(.,'${key_fieldLabel}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TEXT_CONFIRMATION_FIELD</td>
+	<td>//input[@type='text' and contains(@name, 'confirmationField') and contains(@name,'${key_fieldName}')]</td>
```

Comment:
@brunofariasdeo, can we use `@name='${key_fieldName}` instead?

Reply:
I don't think so. 
For example, the confirmation field name is _"_com_liferay_dynamic_data_mapping_form_web_portlet_DDMFormPortlet_ddm$$Text87428451$pMBfo3vG$0$$en_USconfirmationField"_ , so this change would not work. Do you agree?

Reply:
Makes sense to me, @brunofariasdeo. :wink: Thanks.